content/guides/testing/introduction.md

```diff
@@ -218,6 +220,34 @@ export const runnerHooks: Required<Pick<Config, 'setup' | 'teardown'>> = {
 }
 ```
 
+#### Truncate database after each run cycle
+
+An alternative to the above approach is to truncate all table in the database after each run cycle instead of roll it back. This can be done by registering the `TestUtils.db().truncate()` hook within the `tests/bootstrap.ts` file.
+
+```ts
+// title: tests/bootstrap.ts
+export const runnerHooks: Required<Pick<Config, 'setup' | 'teardown'>> = {
+  setup: [
+    () => TestUtils.ace().loadCommands(),
+    // highlight-start
+    () => TestUtils.db().truncate()
+    // highlight-end
+  ],
+  teardown: [],
+}
+```
+
+Now, before running your tests, the hook will migrate the database if necessary. After the tests are run, all the tables in your database will be kept but truncated. 
+
+So next time you run your tests, your database will be empty, but will not need to be migrated again. This may be a better approach and will save you some time if you have a lot of migrations.
```

Comment:
```suggestion
So next time you run your tests, your database will be empty but will not need to be migrated again. This may be a better approach and will save you some time if you have a lot of migrations.
```